docs/development/UX/strategy.md

```diff
@@ -0,0 +1,48 @@
+# UX strategy
+
+## Comfort of use
+
+The device should be pleasant to use. Users would have to hold the phone and the Kampela as a stack and operate on it simultaneously. We should not add unnecessary difficulties on top of that. Things like metadata management should happen automatically and without users' help.
+
+Stack configuration must be researched and determined. Possible options are:
+- flip: Kampela screen is on the opposite side of the phone screen
+- slide: Kampela screen is visible either under or over the phone screen
+- book: Kampela screen is to the side of the phone screen
+
+UI decisions on device shape and handling should be made based on live user tests with device mock models or prototypes.
+
+## Friction
+
+Similar to approach taken in Parity Signer development, certain allowed actions that require extra attention should result in friction. All transactions must be read and explicitly confirmed before accessing the secret storage. Removal/change of trust sources should be sufficiently difficult to stop users from doing it mindlessly.
+
+## Forbidden actions
+
+Certain actions should be forbidden altogether even if complete device replacement is the only alternative.
+
```

Comment:
As for the replacement:
Here I see 2 options - in all three cases, the issue must be resolved without additional disassembly, otherwise it will be necessary to maintain an additional staff with appropriate logistics.
1 Replacement due to malfunction
2 Replacement due to discredit
In both cases, it is advisable to use the same protocol. The protocol should provide for remote registration in the system of the very fact of using the protocol and remote disposal (the technical possibilities of self-destruction of either the device or its contents should be considered, excluding the very possibility of using the contents of the device).

Reply:
The offline nature of the Kampela makes any remote actions impossible. We would have to rely on difficulty of secret extraction from Kampela and on user's vigilance to deal with possible breach in case the device was lost.

In all other scenarios user should safely dispose of device. We would have no tracking of the devices state, again, for privacy and due to our inability to effectively track air-gapped and powered-off for the most of the time Kampelas.

Here by "replacement" I mean that if user wants to perform forbidden actions, they might need to buy a new Kampela and destroy the old one.